nginx_vts_exporter.go

```diff
@@ -144,7 +150,7 @@ type Exporter struct {
 	serverMetrics, upstreamMetrics, cacheMetrics map[string]*prometheus.GaugeVec
 }
 
-func newServerMetric(metricName string, docString string, labels []string) *prometheus.GaugeVec {
+func newServerMetric(metricName string, docString string, labels []string, namespace string) *prometheus.GaugeVec {
```

Comment:
This solves the problem when you want to run the vts-nginx-exporter on different Nginx instances and you have only one Prometheus scrapper.